deepmd/loss/ener.py

```diff
@@ -384,4 +384,169 @@ def print_on_training(self,
         return print_str   
 
 
+class EnerForcesMaskLoss(Loss):
+    """
+    EnerForcesMaskLoss is a concise version of the loss function for DP models.
+    Forces Difference with be multiplied by a `mask4force_matrix: [#frame, #atoms * 3]`.
+    User can specifiy on the forces on which atoms to be fitted by setting `mask4force_matrix`.
 
+    Parameters
+    ----------
+
+    """
+    def __init__(self,
+                 starter_learning_rate : float, 
+                  start_pref_e : float = 0.02,
+                  limit_pref_e : float = 1.00,
+                  start_pref_f : float = 1000,
```

Comment:
`1000` is indeed an `int` but not `float`.

```python
>>> type(1000)
int
```

Reply:
OK, I will fix this.